app/views/application/tips.html.erb

```diff
@@ -0,0 +1,85 @@
+
+Take Notes. Try writing a list of the birds you see, and where and when you saw them. If you keep a logbook, over time, you'll be able to anticipate the movements of birds during migrations. Some birders like to keep a life list, recording every species they have seen in the wild.
+
+Practice! You don't have to actively be looking for birds to practice. Take note of the birds you see and hear on your walk to work or school, while you're looking out your kitchen window, or while you're doing other activities outdoors. You might notice other interesting things, too!
+
+</p>
+<center>
+<h2 style="padding:10px">
+Ethically Birding
+</h2>
+</center>
+
+  <ul style="padding-left:200px">
+    <li>
+      Be aware of sensitive and threatened species that might be vulnerable to disturbance.
+    </li>
+    <li>
+      Do not share nest locations of sensitive species except with appropriate wildlife officials or conservation scientists.
+    </li>
+    <li>
+      Stay at a distance where you are not agitating birds or modifying their behavior, especially near nests.
+    </li>
+    <li>
+      Leave dogs at home or on a leash if in an area with ground nesting birds. Shorebirds, such as Snowy Plovers, that nest on beaches are especially vulnerable to loose dogs.
+    </li>
+    <li>
+      Limit use of pishing and playback — Do not use in heavily birded areas or for sensitive species.
+    </li>
+    <li>
+      Drive slowly and carefully.
+    </li>
+    <li>
+      Stay on designated trails, do not trample vegetation.
+    </li>
+    <li>
+      If leading a birding group, be aware of group size and make sure it is not so large that it is damaging the habitat or interfering with others using the same area.
+    </li>
+    <li>
+      Make sure all group members are aware of and practicing ethical birding guidelines
+    </li>
+    <li>
+      Respect private property rights, only enter with express permission.
+    </li>
+    <li>
+      Follow all traffic rules, drive the speed limit, don’t park on the road, keep eyes on the road.
+    </li>
+    <li>
+      Don’t point binoculars towards other people or directly into yards or windows.
+    </li>
+    <li>
+      Always be polite and courteous to non-birders you encounter, share your knowledge when appropriate.
+    </li>
+    <li>
+      If birding with others, be respectful of the ability of all group members, be encouraging and share your knowledge with beginners.
+    </li>
+  </ul>
+<p style="padding-left:200px">
+  Sources:
+  https://ebird.org/pnw/news/birding-ethics-guidelines-for-respecting-birds-their-habitat-and-each-other <br>
```

Comment:
Seems like this should perhaps be a ul with li and a tags? Be nice for user to be able to click them. Or are they intentionally like this?